src/main/java/peoplesoft/commons/core/GuiSettings.java

```diff
@@ -1,17 +1,21 @@
 package peoplesoft.commons.core;
 
 import java.awt.Point;
-import java.io.Serializable;
 import java.util.Objects;
 
+import com.fasterxml.jackson.annotation.JsonCreator;
+import com.fasterxml.jackson.annotation.JsonProperty;
+
 /**
  * A Serializable class that contains the GUI settings.
  * Guarantees: immutable.
  */
-public class GuiSettings implements Serializable {
+public class GuiSettings {
+    public static final double MIN_WIDTH = 1250;
```

Comment:
i've checked them out, they seem to work fine even at 1250

is it fine to leave it at that then? because iirc the min requirement is that it'll work on a 1280x720 screen, and you'll need to account for window decorations which will give us slightly less than 1280 width

Reply:
we may also want to consider changing the proportions of some of the columns (e.g. in the Job list, shrinking the Duration column slightly and giving more space to the Done column) so that we can make it even narrower without the header text truncating

Reply:
Okay will make an issue to be addressed in my ResultDisplay PR